doc/api/stream.md

```diff
@@ -1493,6 +1493,11 @@ changes:
     pr-url: https://github.com/nodejs/node/pull/18438
     description: >
       Add `emitClose` option to specify if `'close'` is emitted on destroy
+  - version: TBD
```

Comment:
is the value we put here TBD? I never remember.

Reply:
`REPLACEME` :)

Reply:
Ah right! Thanks, will fix